source/setup/pc_setup.rst

```diff
@@ -71,3 +71,6 @@ from the host machine to the VM. Follow these instructions in VirtualBox.
         #. Enable Network Adapter
         #. Attached to Bridged Adapter
         #. Name should be your wired adapter
+        
+.. note::
+    In case you're unable to connect to a robot (i.e. `ssh <robotname>` doesn't work), try installing [Wireshark](https://www.wireshark.org/download.html) and monitoring the packets to diagnose the issue. 	Also try pinging by the expected IP address of the robot, `ping <robotname>.local`, and checking the output of `avahi-browse -av`.
```

Comment:
<robotname>.local should be tried before the IP address.  if ssh doesn't work, you should only look at ping/avahi if the error message is like `Name or service not known`

Reply:
oh, okay. I guess I create a new PR with this changes ?

Reply:
up to you to decide if it's worth your time.  you may be the next person to start helping others with network issue 😁

Reply:
I don't mind. I just wanted to confirm if this is how it's supposed to be done.

Reply:
It's easier to ask forgiveness than it is to get permission

You can make changes directly in master if you think that is best.  There's no "how it's supposed to be done".

You may also want to write that people should ask for help at some point.  No point banging your head against the wall if someone knows that particular robot's wifi doesn't work.